.vscode/settings.json

```diff
@@ -11,5 +11,6 @@
             "source.fixAll": "explicit",
             "notebook.source.organizeImports": "explicit"
         }
-    }
+    },
+    "python-envs.pythonProjects": []
```

Comment:
Why is this needed? And are separate instructions needed for VS code to recognize the uv environment?

Reply:
I am not sure. I added it because VSCode kept adding it. In general uv/venv detection seems to work better out of the box than pixi. Though vscode-python and vscode-python-environments is changing so much that I think we should wait and see a little.